### ISSUE TITLE
'type' object is not subscriptable

### ISSUE THREAD

**Issue by boboliiii** [Godofcoffe/FisherMan#3]
Hi there,  excited to try the program.

(on a windows 7 machine) I get the following error:

Traceback (most recent call last):
  File "fisherman.py", line 344, in <module>
    def scrape(brw: Firefox, items: list[str]):
TypeError: 'type' object is not subscriptable

I'm a bit too much of a noob to figure that one out, from what little I could gather it's probably the list[str] part, maybe needs to be assigned ahead of time or something.. Not sure

Thanks for any help :)

**Comment by Godofcoffe** [comment]
This is a bit of a vague error, can you tell me what version of python you are using and what argument you used in fisherman.py ?

**Comment by boboliiii** [comment]
Thanks for your reply.

I am using python 3.8.8, and I ran it without any arguments (was just finished installing libraries that I was missing).. So I just ran "py fisherman.py" from the MSDOS shell.

**Comment by Godofcoffe** [comment]
I got it, I didn't handle the exception about missing arguments, I will make a note of that and fix it in the next patch, until then always use it with an argument, if you want to see how the tool works, use --help.

**Comment by Godofcoffe** [comment]
And if you are lost on how to use it, take a look at the README.

**Comment by boboliiii** [comment]
Hey there, thanks again.

I tried to run the program with arguments, and have the same error.. I did --help and also -u http://www.facebook.com/someusername/ and both of them raised the same error, "type object is not subscriptable".

Dunno if this helps, but I've sucessfully run other python scripts with selenium, so I don't suspect my python environment is lacking (unless perhaps the versions are not the same as the test environment).
